packable/packable/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.8.2 - 2023-09-07
+
+### Added
+
+- `BTreeSet` impls and `BTreeSetPrefix`;
+
```

Comment:
This PR https://github.com/iotaledger/common-rs/pull/67 forgot to update changelog, can you add an, entry please ?

Reply:
whoops, done